Arrays/chunk.js

```diff
@@ -0,0 +1,24 @@
+const {isValidArray} = require('../input_validation');
+
+function chunk(array, size = 1) {
+  let res = [];
```

Comment:
```suggestion
const res = [];
```
it's better to use const instead let for objects in js